### ISSUE TITLE
1.4.0 invoice header is expecting string[] but demo site given as object[]

### ISSUE THREAD

**Issue by ranraj** [edisonneza/jspdf-invoice-template#22]
The types of 'invoice.header' are incompatible between these types.
```
    Type '({ title: string; style: { width: number; }; } | { title: string; style?: undefined; })[]' is not assignable to type 'string[]'.
```

This source header definition is expecting string[]
[Source code (zip)](https://github.com/edisonneza/jspdf-invoice-template/archive/refs/tags/1.4.0.zip)
Snippet from above source
```
 invDate?: string,
 *       invGenDate?: string,
 *       headerBorder?: boolean,
 *       tableBodyBorder?: boolean,
 *       header?: string[],
 *       table?: any,
 *       invTotalLabel?: string,
 *       invTotal?: string,
```
I get the exception for this 
```
header: [
          {
            title: "#",
            style: {
              width: 10
            }
          },
          {
            title: "Title",
            style: {
              width: 30
            }
          },
          {
```

**Comment by edisonneza** [comment]
Will change it ASAP.

Thanks!

**Comment by edisonneza** [comment]
It's been fixed in 1.4.2
